web/js/tasks.js

```diff
@@ -1318,6 +1318,13 @@ Ext.onReady(function(){
 
     summaryStore.load();
 
+    function isUnToched(taskRecord) {
+        if(taskRecord.get('text') == '' && taskRecord.get('descripton') == '' && taskRecord.get('initTime') == ''
```

Comment:
`description` is not a field of `taskRecord`. Besides, there was a typo :D

I think it's enough with `text`, `initTime` and `projectId`.